scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala

```diff
@@ -76,12 +76,16 @@ object ScaladocParser {
   private def mdCodeBlockIndent[_: P] = hspace.rep(max = 3)
   private def mdCodeBlockFence[_: P] = "`".rep(3) | "~".rep(3)
 
-  private def mdCodeBlockParser[_: P]: P[MdCodeBlock] = P {
-    (startOrNl ~ mdCodeBlockIndent.! ~ mdCodeBlockFence.!).flatMap { case (indent, fence) =>
+
+  private def mdCodeBlockParser[_: P](indent: Int = 1): P[MdCodeBlock] = P {
+    def indentParser = 
+      hspace.rep(min = indent, max = indent + 3)
```

Comment:
so now default is from 1 to 4. before (line 76) was from 0 to 3. is that correct? do we have the right boundary tests?

Reply:
Fixed.

Initially I noticed that `listParser` requires at least one space at the beginning and decided the same rule should be applied for mdCode but I wasn't right. Also changed it for `listParser` and added some tests - scaladoc parser allows list/mdcode/tables without leading space.